ink/yuki/punishment_edging

```diff
@@ -258,21 +262,20 @@ TODO Could add a longer check here to keep the player groveling, should make it
 +[pose:puppy_yes]
   /perform react_laugh amused 60
   ~ addObey()
-    Goodie! But remember, no cumming.
+    Good! But remember, no matter how good it feels, you do NOT have permission to cum.
```

Comment:
I actually like how the line was before. I know you're explaining more, but the short clipped childish lines are more in keeping with Yuki's youthful enthusiasm. :)

Reply:
Reverted :)